src/state/reducers/mentors.ts

```diff
@@ -59,31 +59,16 @@ export const reducer: automaton.Reducer<State, actions.Action> = (
   }
 };
 
-export const skillReducer: automaton.Reducer<string[], actions.Action> = (
-  state = [],
-  action,
-) => {
-  switch (action.type) {
-    case 'skillFilter/toggled':
-      return state.includes(action.payload.skillName)
-        ? state.filter(skill => skill !== action.payload.skillName)
-        : state.concat(action.payload.skillName);
-    case 'skillFilter/reset':
-      return [];
-    default:
-      return state;
-  }
-};
-
-export const getSelectedSkills = (state: types.AppState) => state.skillFilter;
+export const getSelectedSkills = (state: types.AppState) =>
+  state.filterMentors.skillFilter;
```

Comment:
Instead of using this selector in the files, you could use the selector **selectSearchParams** from filterMentors.ts

Then this selector could be removed